README.md

```diff
@@ -145,10 +145,13 @@ $ sam deploy \
 Build the layer by:
 
 1. Installing a Docker environment
-1. Running `make`
+2. Running `make`
```

Comment:
The reason this is two "1"s is markdown will automatically convert it to a correctly numbered list. The "1" merely signifies that it's a numbered list, not the specific number that will be used when the markdown is rendered. By keeping everything "1" we don't have to fiddle with ensuring the numbers are correct when we insert new steps in the middle of numbered lists.